packages/lib-content/src/screens/Home/DefaultHome/DefaultHome.js

```diff
@@ -21,11 +29,24 @@ export default function DefaultHome() {
         align='center'
         background={{ dark: 'dark-3', light: 'neutral-6' }}
         width='min(100%, 90rem)'
-        pad={{ horizontal: 'medium' }}
+        pad={{ horizontal: 'large' }}
       >
         <MaxWidthContent>
           <Introduction />
         </MaxWidthContent>
+        <FeaturedProjects />
+        <Researchers />
+        <SpacedHeading
+          level={2}
+          size='1.5rem'
+          color={{ light: 'neutral-1', dark: 'accent-1' }}
+          textAlign='center'
+          fill
+        >
+          {t('Home.DefaultHome.headings.four')}
+        </SpacedHeading>
+        <SubHeading>{t('Home.DefaultHome.subheadings.four')}</SubHeading>
```

Comment:
SubHeading font size needs a bit of adjustment to adapt to smaller viewport widths. Will do in a separate PR because SubHeading is used throughout the About Zooniverse pages too.